README copy.md

```diff
@@ -1 +0,0 @@
-# VTube
```

Comment:
Please add a descriptive readme file.

- [ ] A good README helps your project to stand out from other projects and should be as good as your project itself.
- [ ] It’s the first thing to notice while encountering your project, so it should be pretty brief but detailed.

Reply:
- [ ] Great job adding the second review. The key to a good code review is to be kind 😄 , polite 🥲 , and offer suggestions, not commands 🧑‍🏭 .